### ISSUE TITLE
Colour contrast in blockquote style is not sufficient

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/docs#187]
![image](https://github.com/localgovdrupal/docs/assets/19519457/5799c032-e1e5-48b3-953a-6405faee51ea)

2.84:1, should be at least 4.5:1 for WCAG 2.1 AA
